src/components/Icon/icons/Google.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import iconColorsMap from './colors';
+
+const Google = ({ fill, ...props }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    {...props}
+  >
+    <g fill="none" fillRule="evenodd" transform="translate(3 3)">
+      <path
+        fill={fill}
```

Comment:
I think these need to be `{fill[0]}` `{fill[1]}` etc since it's an array

Reply:
done